src/util/bootstrapVariables.ts

```diff
@@ -1,40 +1,6 @@
-const style = getComputedStyle(document.body)
+import themeColors from '../styles/bootstrap-theme/colors.module.scss'
```

Comment:
This import can be changed from `themeColors` to `bootstrapVariables`, eliminating the extra variable declaration on line 3, without any apparent issues. This cold make the file more concise, but at the expense of the explicit line from `$theme-colors` to `themeColors` to `bootstrapVariables` that exists with the code as it is.
Do you have a preference?

Reply:
Whats missing is the type declarations to validate the colors we try to extract from this. Aka, we'll get a warning if we try to use `bootstraVarables.bubbleGum` in our code because it knows which keys it has and bubbleGum isn't one of them.

This file could also do the work of camelCasing the kebab-cased keys from the css.